src/components/Nav/Desktop/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { useRef } from "react"
+import { useRouter } from "next/router"
+import { useTranslation } from "next-i18next"
+import { BsTranslate } from "react-icons/bs"
+import { MdBrightness2, MdWbSunny } from "react-icons/md"
+import {
+  Button,
+  HStack,
+  Icon,
+  MenuButton,
+  Text,
+  useColorModeValue,
+  useDisclosure,
+  useEventListener,
+} from "@chakra-ui/react"
+
+import { IconButton } from "@/components/Buttons"
+import LanguagePicker from "@/components/LanguagePicker"
+
+import { DESKTOP_LANGUAGE_BUTTON_NAME, NAV_PY } from "@/lib/constants"
+
+function DesktopNavMenu({ toggleColorMode }: { toggleColorMode: () => void }) {
```

Comment:
@pettinarip Arrow function with extracted props for consistency?

```suggestion
type DesktopNavMenuProps = {
  toggleColorMode: () => void
}

const DesktopNavMenu = ({ toggleColorMode }: DesktopNavMenuProps) => {
```

Reply:
Nice, adding. I forgot to refactor this one.